packages/cli/src/commands/project/rm.ts

```diff
@@ -38,6 +38,10 @@ export default async function rm(client: Client, args: string[]) {
       client.output.error('No such project exists');
       return 1;
     }
+    if (isAPIError(err) && err.status === 403) {
+      client.output.error(err.message);
+      return 1;
+    }
```

Comment:
This is for `vc rm` but I think we need a similar check in `vc link` and other commands

Reply:
`vc link` looks fine to me:

<img width="1415" alt="image" src="https://github.com/vercel/vercel/assets/2198240/6b179303-333a-4cc4-8fd6-b97227fffeea">